src/index.html

```diff
@@ -6,9 +6,80 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">    <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+
+    <header class="header">
+
+      <a href="#" class="header__image">
+        <img src="images/logo.png" alt="|MOYO|">
+      </a>
+
+      <nav class="nav">
+
+        <ul class="nav__list">
+
+          <li class="nav__item first">
```

Comment:
Add empty lines between multiline sibling blocks of HTML. But don't add empty lines between parent and child elements. 
```suggestion
    <header class="header">
      <a href="#" class="header__image">
        <img src="images/logo.png" alt="|MOYO|">
      </a>

      <nav class="nav">
        <ul class="nav__list">
          <li class="nav__item first">
```